db/migrate/20160412133816_create_categories.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCategories < ActiveRecord::Migration
+  def change
+    create_table :categories do |t|
+      t.string :title, null: false
+      t.string :slug, index: true
```

Comment:
I was thinking of this when waking up haha 😄 I changed my mind during the development of this one and realised I could have both null false and db uniqueness for slug as well.